tests/integration/shell/test_matcher.py

```diff
@@ -102,25 +102,24 @@ def test_compound_pillar(self):
 
     @skipIf(True, "This test is unreliable. Need to investigate why more deeply.")
     @flaky
-    def test_coumpound_pillar_pcre(self):
+    def test_compound_pillar_pcre(self):
         data = self.run_salt("-C 'J%@knights%^(Lancelot|Galahad)$' test.ping")
         self.assertTrue(minion_in_returns("minion", data))
         self.assertTrue(minion_in_returns("sub_minion", data))
-        # The multiline nodegroup tests are failing in develop.
-        # This needs to be fixed for Fluorine. @skipIf wasn't used, because
-        # the rest of the assertions above pass just fine, so we don't want
-        # to bypass the whole test.
-        # time.sleep(2)
-        # data = self.run_salt("-C 'N@multiline_nodegroup' test.ping")
-        # self.assertTrue(minion_in_returns('minion', data))
-        # self.assertTrue(minion_in_returns('sub_minion', data))
-        # time.sleep(2)
-        # data = self.run_salt("-C 'N@multiline_nodegroup not sub_minion' test.ping")
-        # self.assertTrue(minion_in_returns('minion', data))
-        # self.assertFalse(minion_in_returns('sub_minion', data))
-        # data = self.run_salt("-C 'N@multiline_nodegroup not @fakenodegroup not sub_minion' test.ping")
-        # self.assertTrue(minion_in_returns('minion', data))
-        # self.assertFalse(minion_in_returns('sub_minion', data))
+
+    def test_compound_nodegroup(self):
+        data = self.run_salt("-C 'N@multiline_nodegroup' test.ping")
+        self.assertTrue(minion_in_returns("minion", data))
+        self.assertTrue(minion_in_returns("sub_minion", data))
+        time.sleep(2)
```

Comment:
Does this need to be here?  I just kept what was commented out.  Does/did this make the test less flaky?